web/src/hash_util.ts

```diff
@@ -241,7 +240,7 @@ export function parse_narrow(hash: string[]): NarrowTerm[] | undefined {
         const operand = decode_operand(operator, raw_operand);
         terms.push({
             negated,
-            operator: operator.toLowerCase(),
+            operator,
```

Comment:
Why is this change part of the revert? I didn't see it in the original commit.

Reply:
It should have been part of the second commit but ended up being a part of the [next one](https://github.com/zulip/zulip/pull/36554/commits/a69b4b62f8f7d675c3f64b4fa3cf849d2ddc6f8f#diff-0f8c07bbe1a5945bcd936f6554c80f13e33b069d4a106205ec6f7d9456268360L239) since I didn't identify that change correctly when I separated out the commits.